.gitignore

```diff
@@ -7,3 +7,4 @@
 /build
 /docs/_build
 /src/urllib3/_sync
+coverage.xml
```

Comment:
I think we should also add few more things in our .gitignore file, I have found this [website](https://www.gitignore.io) to generate .gitignore files, maybe we can take a look at it and see what we need and we might have missed.

Reply:
Rest all looks fine to me.

Reply:
Okay I removed the .gitignore change. Not sure about gitignore.io, it contains a lot of stuff we don't need. I guess we want to open a PR in urllib3/urllib3 for this kind of changes anyway?